salt/haproxy/haproxy.yaml.jinja

```
@@ -20,7 +20,7 @@ spec:
       operator: "Exists"
   containers:
     - name: haproxy
-      image: {{ salt.caasp_registry.base_image_url() }}/haproxy:1.6.0
+      image: {{ salt.caasp_registry.base_image_url() }}/haproxy:1.6.11
```

I think it would be preferable to have a pillar in [params.sls](https://github.com/kubic-project/salt/blob/master/pillar/params.sls) with something like:

```YAML
haproxy:
  image: haproxy:1.6.11
```
and then use here:

```suggestion
      image: {{ salt.caasp_registry.base_image_url() }}/{{ pillar['haproxy']['image'] }}
```

instead of this hardcoded version of the image.

---

follow-up issue created